doc/CommitStyle.md

```diff
@@ -0,0 +1,19 @@
+# Commit message style #
+
+To make the commit messages consistent, the following style of messages is recommended:
+```
+  <type>: [<section>] Your message here [Fix #<num>] (credits) ```
+with
+  * **`<type>`** Type of change, see the list below. **Always include this.**
+  * **optional `[<section>]`** if the commit is in a special section, for example graphics or build system.
+  * **optional `[Fix #<num>]`** if the commit solves an issue, mention it. This means that the issue integration on Github will automatically close it.
```

Comment:
Also, a future reader of the commit can retrieve the issue and check what was the cause for the fix